Sources/Recycling/ListViews/UserAds/Cell/UserAdsListViewSeeAllAdsCell.swift

```diff
@@ -43,8 +43,8 @@ public class UserAdsListViewSeeAllAdsCell: UITableViewCell {
         addSubview(seeAllAdsButton)
 
         NSLayoutConstraint.activate([
-            seeAllAdsButton.heightAnchor.constraint(equalTo: heightAnchor, multiplier: 0.6),
-            seeAllAdsButton.widthAnchor.constraint(equalTo: widthAnchor, multiplier: 0.8),
+            seeAllAdsButton.heightAnchor.constraint(equalTo: heightAnchor, multiplier: 1),
```

Comment:
Guess the multiplier is not really needed here anymore. :-)

Reply:
Nice catch, fixed :)